docs/All/internships.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Internships
+tags:
+  - CAS
+  - Tandon
+  - Planning
+---
+
+# Interships
+
+## Why
+
+Internships are a great way to get work experience. Not only do they make you a more competitive applicant when looking for a job after graduation, most companies will offer return offers to previous interns.
+
+## When
+
+One of the biggest mistakes students make is when to apply for internships. Applications open very early, generally in the fall (August/September/October) for summer internships. Most of the bigger companies will finish their recruiting by Christmas/New Years. You should start applying as soon as possible.
+
+Many of the larger companies will have dedicated internships for freshman/sophomore such as [Google's STEP program](https://buildyourfuture.withgoogle.com/programs/step/). These programs are a great way to get your foot in the door at a big company.
+
+## How
+
+You should have an up to date resume ready. Your resume should be succinct and at most one page. Other resources like a LinkedIn profile or GitHub account can also be helpful, but are not required for all applications.
```

Comment:
could be added to resume section: https://www.nyu.edu/students/student-information-and-resources/career-development-and-jobs/prepare-for-the-job-search/resumes-and-cover-letters.html